cmd/build.go

```diff
@@ -114,16 +115,23 @@ func newBuildCmd() *cobra.Command {
 				logrus.Fatalf("failed to parse meta %s, meta must be formated with JSON: %v", string(metaJSON), err)
 			}
 
-			homedir, err := homedir.Dir()
+			cwd, err := os.Getwd()
 			if err != nil {
 				logrus.Fatal(err)
 			}
 
-			sdlocalDir := filepath.Join(homedir, ".sdlocal")
-			cwd, err := os.Getwd()
-			if err != nil {
-				logrus.Fatal(err)
+			var configBaseDir string
+
+			if useLocalConfig {
+				configBaseDir = cwd
+			} else {
+				configBaseDir, err = homedir.Dir()
+				if err != nil {
+					logrus.Fatal(err)
+				}
 			}
```

Comment:
Is it better to write the following?

```suggestion
			configBaseDir := cwd

			if !useLocalConfig {
				configBaseDir, err = homedir.Dir()
				if err != nil {
					logrus.Fatal(err)
				}
			}
```